build/__dotNetCoreSdkVersions.sh

```diff
@@ -8,3 +8,4 @@ DOT_NET_CORE_31_SDK_VERSION='3.1.412'
 DOT_NET_CORE_50_SDK_VERSION='5.0.302'
 DOT_NET_50_SDK_VERSION='5.0.400'
 DOT_NET_60_SDK_VERSION='6.0.100-preview.7.21379.14'
```

Comment:
DOT_NET_60_SDK_VERSION='6.0.100-rc.1.21458.32'

Reply:
Thank you, updated